spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/jmx/ParentAwareNamingStrategy.java

```diff
@@ -57,8 +57,7 @@ public void setEnsureUniqueRuntimeObjectNames(boolean ensureUniqueRuntimeObjectN
 	@Override
 	public ObjectName getObjectName(Object managedBean, String beanKey) throws MalformedObjectNameException {
 		ObjectName name = super.getObjectName(managedBean, beanKey);
-		Hashtable<String, String> properties = new Hashtable<>();
-		properties.putAll(name.getKeyPropertyList());
+		Hashtable<String, String> properties = new Hashtable<>(name.getKeyPropertyList());
```

Comment:
I find personally the previous version more readable: nesting the method call in the constructor parameter makes it harder to read.

Reply:
I think the previous version of the code will cause the resize of the hash table, so i commit these change.Is my thought something wrong?